Dockerfile

```diff
@@ -8,6 +8,10 @@ RUN npm ci
 COPY . .
 RUN npm run build
 
+# Development stage
+FROM builder AS dev
+EXPOSE 5173
```

Comment:
This is also unneeded because you're already configuring this at your compose file via port.

This would only be appropriate if you're planning to run the image via `docker` instead of `compose`